### ISSUE TITLE
Where could we get android aar binaries for quack?

### ISSUE THREAD

**Issue by abner** [koush/quack#2]
Where could we get the aar for quack.

I tried to get it through jitpack.io but the build is failing. I  failed to compile it by myself.

Do you have the aar published somewhere?

I would like to give quack a try. Thanks in advance!

**Comment by koush** [comment]
Sorry, no aar are available yet, but probably soon. I didn't want to release something that was still in heavy flux. It's mostly stabilized now (as evidenced by the slowing of commits), and it's been deployed to a couple million users in my apps, so I'm feeling good about it.

**Comment by koush** [comment]
quack should build for you though.

just make sure you have my quickjs fork also checked out side by side for the ndk build.

```
cd your-project-dir
git clone https://github.com/koush/quack
git clone https://github.com/koush/quickjs
```

then

```
include ':quack:quack-android'
```

**Comment by mzgreen** [comment]
Hey @koush any updates on publishing this library?

**Comment by koush** [comment]
Yeah, this is stable at this point. Wanted to get the debugger done (windows implementation hasn't been merged) and some examples for that. https://github.com/koush/vscode-quickjs-debug

Within the next couple weeks hopefully.
